runner/src/main.rs

```diff
@@ -967,6 +968,24 @@ fn run_cargo_udeps() -> Step {
     }
 }
 
+fn run_cargo_clean() -> Step {
+    Step::Multiple {
+        name: "cargo clean".to_string(),
+        steps: crate_manifest_files()
+            .map(|manifest_path| {
+                let manifest_path = manifest_path.to_str().unwrap();
+                Step::Single {
+                    name: manifest_path.to_string(),
```

Comment:
If you first map `manifest_path` to string, then here you can call `clone` here.

Reply:
done!